api/api.go

```diff
@@ -114,6 +114,9 @@ func main() {
 			if userResponse.Error != nil {
```

Comment:
- [ ] Merge #108 

Looks great! Thanks for adding the screenshots. 
I think we should merge #108 first so I can deploy the changes on AWS and make sure the Public DNS doesn't change.

Reply:
What a fantastic idea! I'll get right on that!

Reply:
That PR has been closed, and the branch is now merged!!!